jcli/src/jcli_lib/certificate/sign.rs

```diff
@@ -244,3 +252,43 @@ where
     let sig = PoolOwnersSigned { signatures: sigs };
     Ok(to_signed_certificate(payload, sig))
 }
+
+pub(crate) fn update_proposal_sign<P: Payload>(
+    update_proposal: UpdateProposal,
+    keys_str: &[String],
+    builder: TxBuilderState<SetAuthData<P>>,
+) -> Result<SignedCertificate, Error> {
+    if keys_str.len() > 1 {
+        return Err(Error::ExpectingOnlyOneSigningKey {
```

Comment:
Should we just not accept multiple singing keys in this function and reject them early?